Bottom: e84e4098246de459f2119ba124b64602e1747a37
Top:    833072759ca8ec96915295e6d49b069edc9316a1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-12 17:14:18 +0000

Refresh of enable-wextra-and-pedantic

---

diff --git a/configure.ac b/configure.ac
index 53a96ca..10d84b5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1143,7 +1143,7 @@ LIBS="$LIBS $XM_LIBS $DBUS_LIBS $X_LIBS $GLIB_LIBS $GTK_LIBS $DMALLOC_LIBS $GD_L
 # if we have gcc then add -Wall
 if test "x$GCC" = "xyes"; then
 	# see about adding some extra checks if the compiler takes them
-	for flag in -Wall -Wextra -pedantic; do
+	for flag in -Wall -Wextra; do #-pedantic
 		case " ${CFLAGS} " in
 			*\ ${flag}\ *)
 				# flag is already present
